services/php-apache/Dockerfile

```diff
@@ -3,10 +3,11 @@ ENV COMPOSER_NO_INTERACTION=1 \
     COMPOSER_ALLOW_SUPERUSER=1 \
     DOCROOT=/var/www/html
 
-RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev mysql-client && \
+RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev && \
+    (apt-get install -y mysql-client || apt-get install -y mariadb-client) && \
     \
-    docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && \
-    docker-php-ext-install -j$(nproc) gd iconv pdo_mysql && \
+    (docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ || /bin/true) && \
+    (docker-php-ext-install -j$(nproc) gd iconv pdo_mysql || /bin/true) && \
```

Comment:
Don't we want to fail if these extensions fail to build?

Reply:
there's a problem with buster & php7.3 where these will fail to build, don't want that to block the other versions

Reply:
https://github.com/docker-library/php/issues/865